package.json

```diff
@@ -44,22 +44,23 @@
   "productName": "DeltaChat",
   "homepage": "https://delta.chat",
   "dependencies": {
-    "@blueprintjs/core": "^3.4.0",
+    "@blueprintjs/core": "^3.16.2",
     "@blueprintjs/select": "^3.8.1",
     "@mapbox/geojson-extent": "^0.3.2",
     "application-config": "^1.0.1",
-    "array-differ": "^2.0.3",
+    "array-differ": "^3.0.0",
     "class-autobind": "^0.1.4",
     "debounce": "^1.2.0",
     "deltachat-node": "1.0.0-alpha.0",
     "emoji-js-clean": "^4.0.0",
     "emoji-mart": "^2.11.1",
     "error-stack-parser": "^2.0.2",
-    "filesize": "^3.6.1",
-    "immutability-helper": "^3.0.0",
+    "filesize": "^4.1.2",
+    "hallmark": "^0.1.0",
```

Comment:
why did you move hallmark to the normal dependencies?

Reply:
oops - that was not on purpose. I just checked if the vulnerability warnings would vanish without hallmark and readded it wrong